crates/aranya-client/src/client/team.rs

```diff
@@ -399,6 +399,26 @@ impl Team<'_> {
             .map_err(aranya_error)
     }
 
+    /// Add label managing role
+    #[instrument(skip(self))]
+    pub async fn add_label_managing_role(
+        &self,
+        label_id: LabelId,
+        managing_role_id: RoleId,
+    ) -> Result<()> {
+        self.client
+            .daemon
+            .add_label_managing_role(
+                context::current(),
+                self.id,
```

Comment:
Nit: it could be more clear to change the name of the field to `team_id`:
```suggestion
                self.team_id,
```
I had to look up other code to figure out that it's not a device, label, or role ID.
Contextually this may be somewhat obvious so feel free to disregard.